js/classes/player.js

```diff
@@ -37,22 +36,71 @@ class Player extends Character {
     this._inv = [];
     this._attackTimer = null;
 
-    this._icon = "user";
+    this._icon = 'user';
+    this._type = 'player';
 
     //build an icon
-    this._elements["createIcon"] = buildElement(
-      "touch-icon",
-      { class: "player" },
-      this.getInfo
+    this._elements['createIcon'] = buildElement('touch-icon', 
+                                                {class: 'player'}, 
+                                                this.getInfo);
+    this._elements['playerMenu'] = buildElement('player-menu',
+                                                {id: 'player-menu'},
+                                                {_isOpen: false});
+    
+  
+    //attach icon to icon-bar
+    document.querySelector('#icon-bar').appendChild(this._elements['createIcon']);
+    //attach player menu to icon-bar
+    document.querySelector('#icon-bar').appendChild(this._elements['playerMenu']);
+    //append menu items
+    
+    this._elements['playerMenu'].appendChild(buildElement('touch-icon',
+                                                          {class: 'menu-item'},
+                                                          {icon: 'boxes',
+                                                           type: 'inv'}));
+    this._elements['playerMenu'].appendChild(buildElement('touch-icon', 
+                                                          {class: 'menu-item'}, 
+                                                          {icon: 'arrows-alt',
+                                                           type: 'move'}));
+
```

Comment:
this is some strange formatting. could you clean this up before the merge?

Reply:
I don't know how else to format this, but I did change playermenu to a component so it isn't as long any more. Trying to get across that buildElement takes some args. Is there a suggestion for cleaning this formatting?